.gitattributes

```diff
@@ -1,3 +1,5 @@
 borg/_version.py export-subst
 
 *.py diff=python
+docs/usage merge=ours
+docs/man merge=ours
```

Comment:
Shouldn't be, it's listed on the man page as a built-in strategy. But another mistake kept it from working; fixed-- the option does not work like a .gitignore, it only works for files, so a `/*` was missing. Seems to work now.